### PR TITLE
Refactor Code

### PR DESCRIPTION
**Pull request by auhc99**
Changed anime to animes for plural uses
Changed aniList to animeList
Changed addressBook to animeList

### CODE REVIEW

File: docs/tutorials/RemovingFields.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ In `src/test/data/`, data meant for testing purposes are stored. While keeping t
 
 ```json
 {
-  "anime": [ {
+  "animes": [ {
     "name": "Person with invalid name field: Ha!ns Mu@ster",
```
Comment (muhammad-faruq) [inline docs/tutorials/RemovingFields.md:104]: shouldnt this part be changed too? name phone and email should probably be changed to.. name may stay as name but phone and email does not make sense here

File: docs/tutorials/TracingCode.md [hunk 2]
```diff
@@ -202,9 +202,9 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
 
 1. As suspected, `command#execute()` does indeed make changes to the `model` object. Specifically,
    * it uses the `setPerson()` method (defined in the interface `Model` and implemented in `ModelManager` as per the usual pattern) to update the anime data.
```
Comment (muhammad-faruq) [inline docs/tutorials/TracingCode.md:204]: should setPerson be changed here?

File: docs/tutorials/TracingCode.md [hunk 3]
```diff
@@ -231,7 +231,7 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
      * {@code JsonSerializableAddressBook}.
      */
     public JsonSerializableAddressBook(ReadOnlyAddressBook source) {
```
Comment (muhammad-faruq) [inline docs/tutorials/TracingCode.md:233]: Does all these AddressBooks need to change too?

File: src/main/java/seedu/anilist/storage/AnimeListStorage.java [hunk 4]
```diff
@@ -11,36 +11,36 @@
...
+public interface AnimeListStorage {
 
     /**
      * Returns the file path of the data file.
      */
-    Path getAniListFilePath();
+    Path getAnimeListFilePath();
 
     /**
      * Returns AniList data as a {@link ReadOnlyAnimeList}.
```
Comment (muhammad-faruq) [inline src/main/java/seedu/anilist/storage/AnimeListStorage.java:22]: Returns AnimeList data

File: src/main/java/seedu/anilist/storage/JsonAnimeListStorage.java [hunk 5]
```diff
@@ -17,36 +17,36 @@
 /**
  * A class to access AniList data stored as a json file on the hard disk.
```
Comment (muhammad-faruq) [inline src/main/java/seedu/anilist/storage/JsonAnimeListStorage.java:18]: AnimeList

File: src/main/java/seedu/anilist/storage/JsonSerializableAnimeList.java [hunk 6]
```diff
@@ -17,26 +17,26 @@
  * An Immutable AniList that is serializable to JSON format.
  */
 @JsonRootName(value = "anilist")
```
Comment (muhammad-faruq) [inline src/main/java/seedu/anilist/storage/JsonSerializableAnimeList.java:19]: anilist here and on line 17

File: src/test/java/seedu/anilist/model/ModelManagerTest.java [hunk 7]
```diff
@@ -62,14 +62,14 @@ public void setGuiSettings_validGuiSettings_setsGuiSettings() {
 
     @Test
     public void setAddressBookFilePath_nullPath_throwsNullPointerException() {
```
Comment (muhammad-faruq) [inline src/test/java/seedu/anilist/model/ModelManagerTest.java:64]: Address book here

File: src/test/java/seedu/anilist/model/ModelManagerTest.java [hunk 8]
```diff
@@ -37,14 +37,14 @@ public void setUserPrefs_nullUserPrefs_throwsNullPointerException() {
...
-        userPrefs.setAniListFilePath(Paths.get("address/book/file/path"));
+        userPrefs.setAnimeListFilePath(Paths.get("address/book/file/path"));
         userPrefs.setGuiSettings(new GuiSettings(1, 2, 3, 4));
         modelManager.setUserPrefs(userPrefs);
         assertEquals(userPrefs, modelManager.getUserPrefs());
 
         // Modifying userPrefs should not modify modelManager's userPrefs
         UserPrefs oldUserPrefs = new UserPrefs(userPrefs);
-        userPrefs.setAniListFilePath(Paths.get("new/address/book/file/path"));
+        userPrefs.setAnimeListFilePath(Paths.get("new/address/book/file/path"));
```
Comment (muhammad-faruq) [inline src/test/java/seedu/anilist/model/ModelManagerTest.java:47]: file path should be anime list not address/book.. similarly line 40

File: src/main/java/seedu/anilist/storage/JsonSerializableAnimeList.java (hunk 6 above)
Comment (auhc99) [reply src/main/java/seedu/anilist/storage/JsonSerializableAnimeList.java:19]: Changed line 17, but for line 19 I think that refers to the package name. which will be 'anilist'

File: docs/tutorials/TracingCode.md (hunk 3 above)
Comment (auhc99) [reply docs/tutorials/TracingCode.md]: changed

File: docs/tutorials/TracingCode.md (hunk 2 above)
Comment (auhc99) [reply docs/tutorials/TracingCode.md]: changed

File: src/main/java/seedu/anilist/storage/AnimeListStorage.java (hunk 4 above)
Comment (auhc99) [reply src/main/java/seedu/anilist/storage/AnimeListStorage.java]: changed

File: src/main/java/seedu/anilist/storage/JsonAnimeListStorage.java (hunk 5 above)
Comment (auhc99) [reply src/main/java/seedu/anilist/storage/JsonAnimeListStorage.java]: changed

File: src/test/java/seedu/anilist/model/ModelManagerTest.java (hunk 7 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/model/ModelManagerTest.java]: changed

File: src/test/java/seedu/anilist/model/ModelManagerTest.java (hunk 8 above)
Comment (auhc99) [reply src/test/java/seedu/anilist/model/ModelManagerTest.java]: changed

**Pull request merged**